src/admin/engineHeader.php

```diff
@@ -8,17 +8,17 @@
 $localvars  = localvars::getInstance();
 $enginevars = enginevars::getInstance();
 
-recurseInsert("acl.php","php"); 
+recurseInsert("acl.php","php");
 recurseInsert("includes/vars.php","php");
 recurseInsert('includes/functions.php',"php");
 
 recurseInsert('engineIncludes.php');
 
-formBuilder::process();
-formBuilder::ajaxHandler();
+// formBuilder::process();
+// formBuilder::ajaxHandler();
```

Comment:
These two lines prevent all forms from being able to use callbacks.  In one instance we need a callback to check for duplicate items that are only duplicates if they match on multiple fields.  Having these at top level instead of using the in form processing blocks this.  Instead these have been moved to all form definitions so that if needed callbacks can be added.